### ISSUE TITLE
Update allowed values for L in manual

### ISSUE THREAD

**Issue by gcapes** [zan8simone/PPML---Periodically-Patterned-Multi-Layer#5]
This currently shows as `L > 0`, but I think @zan8simone said it should be `L >= 0`.

**Comment by zan8simone** [comment]
correct. L=0 is the case where there are no internal layers, that is, there is only a superstrate and a substrate.
